pkg/reconcilermanager/controllers/reposync_controller.go

```diff
@@ -864,15 +844,21 @@ func (r *RepoSyncReconciler) validateSpec(ctx context.Context, rs *v1beta1.RepoS
 	case v1beta1.OciSource:
 		return validate.OciSpec(rs.Spec.Oci, rs)
 	case v1beta1.HelmSource:
-		if err := validate.HelmSpec(reposync.GetHelmBase(rs.Spec.Helm), rs); err != nil {
-			return err
-		}
-		return validate.CheckValuesFileRefs(ctx, r.client, rs.Spec.Helm.ValuesFileRefs, rs)
+		return validate.HelmSpec(reposync.GetHelmBase(rs.Spec.Helm), rs)
 	default:
 		return validate.InvalidSourceType(rs)
 	}
 }
 
+// checkValuesFileSourcesRefs validates that the ConfigMaps specified in the RSync ValuesFileSources exist and have the
+// specified data key.
+func (r *RepoSyncReconciler) checkValuesFileSourcesRefs(ctx context.Context, rs *v1beta1.RepoSync) status.Error {
```

Comment:
rename validate- instead of check- for consistency with other validation methods

Reply:
done